### PR TITLE
POST a feature collection to the items route

### PR DESCRIPTION
**Pull request by jonhealy1**
**Related Issue(s):** 

- #362 

**Description:**
Allow for the items route to accept a feature collection as well as a single item.

**PR Checklist:**

- [x] Code is formatted and linted (run `pre-commit run --all-files`)
- [x] Tests pass (run `make test`)
- [x] Documentation has been updated to reflect changes, if applicable, and docs build successfully (run `make docs`)
- [x] Changes are added to the [CHANGELOG](https://github.com/stac-utils/stac-fastapi/blob/master/CHANGES.md).

### CODE REVIEW

**Comment by philvarner** [comment]
needs a changelog entry.

File: stac_fastapi/sqlalchemy/stac_fastapi/sqlalchemy/transactions.py [hunk 1]
```diff
@@ -36,6 +36,15 @@ class TransactionsClient(BaseTransactionsClient):
     def create_item(self, model: stac_types.Item, **kwargs) -> stac_types.Item:
         """Create item."""
         base_url = str(kwargs["request"].base_url)
+
+        # If a feature collection is posted
+        if model["type"] == "FeatureCollection":
+            bulk_client = BulkTransactionsClient(session=self.session)
+            return_msg = bulk_client.bulk_item_insert(items=model["features"])
+
+            return return_msg
```
[the remark this answers is not in the document]
Comment (philvarner) [reply stac_fastapi/sqlalchemy/stac_fastapi/sqlalchemy/transactions.py:45]: I'm about to open another PR that will actually change that signature to be Optional[stac_types.Item], to advertise that None is a valid response

[the remark this answers is not in the document]
Comment (jonhealy1) [reply stac_fastapi/sqlalchemy/stac_fastapi/sqlalchemy/transactions.py]: Ok sounds good

File: stac_fastapi/sqlalchemy/stac_fastapi/sqlalchemy/transactions.py [hunk 2]
```diff
@@ -35,11 +35,19 @@ class TransactionsClient(BaseTransactionsClient):
     )
 
     def create_item(
-        self, item: stac_types.Item, **kwargs
+        self, model: Union[stac_types.Item, stac_types.ItemCollection], **kwargs
```
Comment (moradology) [inline stac_fastapi/sqlalchemy/stac_fastapi/sqlalchemy/transactions.py:38]: I've updated this to reflect the fact that the model is *either* `Item` or `ItemCollection`

**Review by moradology — APPROVED** [review]

**Pull request merged**